client/src/components/questions/Question.jsx

```diff
@@ -108,7 +111,7 @@ const Question = ({ questionData, productName }) => {
       <StyledQuestion >
         <h4>{'Q: ' + questionData.question_body}</h4>
         <StyledLinks>
-          Helpful? <u onClick={helpfulClick} style={{cursor: 'pointer'}}>Yes</u> {`(${helpfulCount})`} &nbsp;&nbsp;|&nbsp;&nbsp; <u data-testid={questionData.question_id} onClick={addAnswer} style={{cursor: 'pointer'}}>Add Answer</u>
+          Helpful? <u data-testid={id + 'helpful'} onClick={helpfulClick} style={{cursor: 'pointer'}}>Yes</u> <span>{`(${helpfulCount})`}</span> &nbsp;&nbsp;|&nbsp;&nbsp; <u data-testid={id} onClick={addAnswer} style={{cursor: 'pointer'}}>Add Answer</u>
```

Comment:
Is there a reason you wrapped the helpfulCount in a span?

Reply:
I needed to access it in a test using "getNextElementSibling" from the adjacent <u> element. That was the only way I found to access floating text in the dom like that.